publish.sh

```diff
@@ -11,17 +11,17 @@ git fetch  # Make sure we are up to date with git remote branches
 git add --all
 git config --global user.name "OpenFisca-Bot"
 git config --global user.email "bot@openfisca.org"
-git diff-index --quiet HEAD || git commit --message="[skip ci] Push from openfisca doc"
+git diff-index --quiet HEAD || git commit --message="[skip ci] Update doc"
```

Comment:
We are removing "from openfisca doc" because this update could be started by a change in openfisca-doc or openfisca-core, right? 

> We might be missing some documentation that explains the link between openfisca-core and openfisca-doc (in a futur PR). For exemple, if I remember correctly, [this `Update doc` in openfisca-core GitHub Actions](https://github.com/openfisca/openfisca-core/runs/6936337240?check_suite_focus=true) now produces a [workflow like this one in openfisca-doc CircleCI](https://app.circleci.com/pipelines/github/openfisca/openfisca-doc/659/workflows/4dc763d9-707e-4333-8ddc-7524de5316ce) but the CI doesn't explicit the source of the update.

Reply:
Yes, there are several cases that might trigger such an update. Plus, I am not sure how useful that information is as part of a commit message that updates content.